storage/boltz/crud_test.go

```diff
@@ -624,6 +625,150 @@ func (test *crudTest) testLinkCollection(_ *testing.T) {
 	test.NoError(err)
 }
 
```

Comment:
Add lots of simple tests for different levels of linked entities and combinations of single entity and set entity links